apps/i18n/common/en_us.json

```diff
@@ -42,7 +42,6 @@
   "addStudentsManageMyOwn": "Create and manage my own list of students",
   "addStudentsSyncThirdParty": "Sync my list of students from an existing classroom section in a third party tool",
   "addStudentsToSectionInstructions": "Choose how you want to add your students:",
-  "addStudentsToSectionInstructionsUpdated": "How do you want your students to sign in?",
```

Comment:
Let's leave all the strings we need until we finalize this issue so we don't lose the translations in Crowdin & have to keep re-adding them

Reply:
Oh good point, I'll update that!